features/template/build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Remove once https://youtrack.jetbrains.com/issue/KTIJ-19369 is fixed
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+    id("io.element.android-compose-library")
+    alias(libs.plugins.ksp)
+    alias(libs.plugins.anvil)
+}
+
+android {
+    namespace = "io.element.android.x.features.template"
+}
+
+anvil {
+    generateDaggerFactories.set(true)
+}
+
+dependencies {
+    anvil(project(":anvilcodegen"))
+    implementation(project(":anvilannotations"))
+
+    implementation(project(":libraries:di"))
+    implementation(project(":libraries:core"))
+    implementation(project(":libraries:architecture"))
+    implementation(project(":libraries:matrix"))
+    implementation(project(":libraries:matrixui"))
+    implementation(project(":libraries:designsystem"))
+    implementation(project(":libraries:elementresources"))
+    implementation(project(":libraries:ui-strings"))
```

Comment:
Maybe create  a `fun DependencyHandlerScope.allLibrariesForFeatures()`? (or reuse `allLibraries()`).

I am not sure if we want to be certain that there is no useless dependencies...

Reply:
Would be better to avoid using useless dependencies yes. I prefer to let the user pick the right ones